ui/src/actions/login.js

```diff
@@ -94,8 +95,8 @@ export function getInitialState() {
       fetchQueueState()
         .then((queue) => ({ queue }))
         .catch(notify),
-      fetchBeamInfo()
-        .then((beamInfo) => ({ beamInfo }))
+      sendGetValue('beam', 'beam')
+        .then((beamInfo) => ({ beamInfo: beamInfo.value }))
```

Comment:
Ill try to come up with a way to get rid of this extra '.value' that indirectly comes from the use of PyDantic models for the adapter `get_value`.